tests/libnet/ipv6.go

```diff
@@ -53,3 +53,13 @@ func ConfigureIPv6OnVMI(vmi *v1.VirtualMachineInstance) error {
 
 	return nil
 }
+
+func WithIPv6(loginToFactory console.LoginToFactory) console.LoginToFactory {
+	return func(vmi *v1.VirtualMachineInstance) error {
+		err := loginToFactory(vmi)
+		if err != nil {
+			return err
+		}
+		return ConfigureIPv6OnVMI(vmi)
```

Comment:
Please change `ConfigureIPv6OnVMI` to be a private method.

Reply:
Done